content/includes/nap-waf/config/v5/host-based-nginx-instructions/nginx-plus-centos-9.md

```diff
@@ -0,0 +1,17 @@
+Create NGINX App Protect WAF v5 repository file, named `/etc/yum.repos.d/app-protect-x-plus.repo` with the following contents:
+
+```none
+[app-protect-x-plus]
+name=nginx-app-protect repo
+baseurl=https://pkgs.nginx.com/app-protect-x-plus/centos/9/$basearch/
```

Comment:
Comment only. I see a lot of these `baseurl`s with a specific number. Can't we replace them with $releasever?